### PR TITLE
Bug/sc 23445/make clark logo clickable

### PR DESCRIPTION
**Pull request by shanioffe**
This addresses the story https://app.shortcut.com/clarkcan/story/23445/make-clark-logo-clickable and makes CLARK logo open homepage in new tab when clicked.

### CODE REVIEW

File: .husky/pre-commit
Comment (Cwagne17) [inline .husky/pre-commit:1]: How come this file is being removed?

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npm run lint && npm run test:unit
```
Comment (Cwagne17) [inline .husky/pre-commit:4]: I'd prefer that we remove the test but keep the linting atleast.

**Review by Cwagne17 — COMMENTED** [review]
@mfranz1 and I were discussing. This will have to be redone based off of the non-angular upgrade branch

**Comment by shanioffe** [comment]
Wrong base branch, need to redo this PR

**Pull request closed without merging**